pages/common/jq.md

```diff
@@ -0,0 +1,27 @@
+# jq
+
+> A lightweight and flexible command-line JSON processor.
+
+- Output JSON file:
+
+`cat {{file}} | jq`
```

Comment:
If the file already contains JSON text, what is this going to change ? Just pretty-print ? If yes. we should mention that.